src/pages/Login/Login.jsx

```diff
@@ -1,8 +1,190 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate, Link } from 'react-router-dom';
 import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const navigate = useNavigate();
+
+  const emailReg =
+    /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/;
+  const passwordReg =
+    /^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$/;
+  const isEmailValid = emailReg.test(email);
+  const isPasswordValid = passwordReg.test(password);
+
+  const goToHome = () => {
+    navigate('/');
+  };
+
+  const handleLogin = e => {
+    e.preventDefault();
+    fetch('http://10.58.52.73:3000/signin', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    })
+      .then(response => {
+        return response.json();
+      })
```

Comment:
```suggestion
      .then(response =>  response.json())
```

필수는 아닙니다!!
하지만 이렇게 줄일 수도 있습니다.
함수를 선언할 때 단 한줄이라면 {}와 return을 없앨 수 있어요.